src/common/range.rs

```diff
@@ -416,3 +439,10 @@ fn test_byte_range_spec_to_satisfiable_range() {
 bench_header!(bytes_multi, Range, { vec![b"bytes=1-1001,2001-3001,10001-".to_vec()]});
 bench_header!(custom_unit, Range, { vec![b"other=0-100000".to_vec()]});
 */
+
+#[test]
+fn test_to_satisfiable_range_suffix() {
+    let range = super::test_decode::<Range>(&["bytes=-100"]).unwrap();
+    let bounds = range.satisfiable_ranges(350).next().unwrap();
+    assert_eq!(bounds, (Bound::Included(250), Bound::Unbounded));
+}
```

Comment:
Should we also include this?

```suggestion
}

#[test]
fn test_to_unsatisfiable_range_suffix() {
    let range = super::test_decode::<Range>(&["bytes=-350"]).unwrap();
    let bounds = range.satisfiable_ranges(100).next();
    assert_eq!(bounds, None);
}
```